<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Form{
	background-color:rgb(37, 150, 190);
}
QLabel[statusTip = &quot;this&quot;]{
	color:black;
}
QHBoxLayout{
	background-color:black;
}
QFrame[statusTip = &quot;option&quot;]{

 }

QFrame[statusTip = &quot;option&quot;][enabled = &quot;true&quot;]:hover{
	background-color: rgba(110, 110, 110, 120);
	border-left: 6px solid #000;
 }
QFrame[statusTip = &quot;option&quot;][enabled = &quot;false&quot;]{
	background-color: rgba(110, 110, 110, 120);
	border-left: 6px solid #000;
 }
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons/icons.qrc">:/icons/admin.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;
font: bold;</string>
    </property>
    <property name="text">
     <string>Meher Majdoub</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size : 20px;
font:bold;</string>
    </property>
    <property name="text">
     <string>ADMIN</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>750</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons/icons.qrc">:/icons/setting.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="containor">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>801</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(37, 150, 190)</string>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="stats_frame">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>220</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 50px;</string>
     </property>
     <property name="text">
      <string>STATS</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="products_frame">
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>310</y>
       <width>221</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 50px;</string>
     </property>
     <property name="text">
      <string>products</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="categories_frame">
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>420</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 50px;</string>
     </property>
     <property name="text">
      <string>categories</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="employees_frame">
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>290</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 50px;</string>
     </property>
     <property name="text">
      <string>employees</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sellings_frame">
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>290</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 50px;</string>
     </property>
     <property name="text">
      <string>sellings</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="bills_frame">
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>290</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 50px;</string>
     </property>
     <property name="text">
      <string>BILLS</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>31</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(110,110,110,50)</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>800</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons/icons.qrc">:/icons/shutdown.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>221</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="options">
    <item>
     <widget class="QFrame" name="stats_lbl">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="statusTip">
       <string>option</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="statusTip">
        <string>this</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Stats</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>51</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons/icons.qrc">:/icons/stats.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="stats_btn">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>221</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgba(0,0,0,0);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="products_lbl">
      <property name="statusTip">
       <string>option</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>51</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons/icons.qrc">:/icons/products.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="statusTip">
        <string>this</string>
       </property>
       <property name="whatsThis">
        <string>this</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel:pressed{
color:red;
}
</string>
       </property>
       <property name="text">
        <string>Products</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>1</number>
       </property>
       <property name="buddy">
        <cstring>this</cstring>
       </property>
      </widget>
      <widget class="QPushButton" name="products_btn">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>221</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgba(0,0,0,0);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="categories_lbl">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="statusTip">
       <string>option</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>51</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons/icons.qrc">:/icons/categories.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="statusTip">
        <string>this</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Categories</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="categories_btn">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>221</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgba(0,0,0,0);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="employees_lbl">
      <property name="statusTip">
       <string>option</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>159</width>
         <height>51</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="statusTip">
        <string>this</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Employees</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>51</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons/icons.qrc">:/icons/employee.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="employees_btn">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>221</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgba(0,0,0,0);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="sellings_lbl">
      <property name="statusTip">
       <string>option</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>51</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons/icons.qrc">:/icons/sellings.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="statusTip">
        <string>this</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Sellings</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="sellings_btn">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>221</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgba(0,0,0,0);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="bills_lbl">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="statusTip">
       <string>option</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>51</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons/icons.qrc">:/icons/bill.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="statusTip">
        <string>this</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Billls</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="bills_btn">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>221</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgba(0,0,0,0);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
